packages/swingset-liveslots/src/vatstore-usage.md

```diff
@@ -96,7 +98,7 @@ Virtual objects are held on disk, which makes them suitable for high-cardinality
 
 Durable Kinds are defined just like virtual Kinds, but they use a different constructor (`defineDurableKind` instead of `defineKind`), which requires a "handle" created by `makeKindHandle`. Durable virtual objects can only hold durable data in their `state`.
 
-The KindHandle is a durable virtual object of a special internal Kind. This is the first Kind allocated, so usually it gets Kind ID 1 and the handles get vrefs like `o+d1/${kindHandleID}`.
+The KindHandle is a durable virtual object of a special internal Kind. This is the first Kind allocated, so usually it gets exportID 1 and the kind handles for virtual and durable Kinds get vrefs like `o+d1/${kindHandleID}`.
```

Comment:
Hm, I want to keep referring to KindIDs as such. But it's awkward that they share a numberspace with ExportIDs, and if we only ever referred to them as KindIDs, it would be confusing (I like the emphasis you've added to the export ID connection here).

Maybe say "so it usually gets exportID 1 to use as a KindID, and the .."